experimental/iterators/test/simple-plan-input.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-opt -allow-unregistered-dialect %s
```

Comment:
This seems premature in this revision?

Reply:
I thought checking that the file has MLIR syntax could be useful, no?

Reply:
In practice this is only testing that the core MLIR custom parser and printer form works. 
This is already well tested upstream.